products/ionic.md

```diff
@@ -11,50 +11,66 @@ activeSupportColumn: true
 eolColumn: Maintenance
 releaseColumn: true
 releaseDateColumn: true
+extendedSupportColumn: true
 
 auto:
 -   git: https://github.com/ionic-team/ionic-framework.git
 
+# support(R) = releaseDate(R+1)
 releases:
--   releaseCycle: "6"
-    releaseDate: 2021-12-08
+-   releaseCycle: "7"
+    releaseDate: 2023-03-29
     support: true
     eol: false
+    extendedSupport: true
+    latest: "7.0.1"
+    latestReleaseDate: 2023-04-05
+
+-   releaseCycle: "6"
+    releaseDate: 2021-12-08
+    support: 2023-03-29
+    eol: 2024-09-29
+    extendedSupport: 2024-03-29
```

Comment:
`extendedSupport` ends before `eol` : that does not look good. Should be Sep 29, 2023 according to https://ionicframework.com/docs/reference/support#framework-maintenance-and-support-status.

Reply:
Fixed.